users/views_test.py

```diff
@@ -144,6 +144,69 @@ def test_get_user_by_me(mocker, client, user, is_anonymous, has_orgs):
         }
 
 
+@pytest.mark.parametrize("is_anonymous", [True, False])
+@pytest.mark.parametrize("has_openedx_user", [True, False])
+def test_get_userinfo(mocker, client, user, is_anonymous, has_openedx_user):
```

Comment:
Can you also test for the case where `OpenEdxUser` exists but `edx_username` isn't set?